WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -150,9 +144,70 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
         return controller
     }
 
-    /// Observe the managedObjectContext for changes (likes, saves) and reload the tableView
-    private func observeDisplayContext() {
-        NotificationCenter.default.addObserver(self, selector: #selector(reload(_:)), name: NSNotification.Name.NSManagedObjectContextDidSave, object: managedObjectContext())
+    private func addObservers() {
+        // Observe the managedObjectContext for changes (likes, saves) and reload the tableView
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(reload(_:)),
+                                               name: .NSManagedObjectContextDidSave,
+                                               object: managedObjectContext())
+
+        // Listens for when the reader manage view controller is dismissed
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(manageControllerWasDismissed(_:)),
+                                               name: .readerManageControllerWasDismissed,
+                                               object: nil)
+    }
+
+    @objc func manageControllerWasDismissed(_ notification: Foundation.Notification) {
+        shouldForceRefresh = true
+        self.displaySelectInterestsIfNeeded()
+    }
+}
+
+// MARK: - Select Interests Display
+private extension ReaderCardsStreamViewController {
+    func displaySelectInterestsIfNeeded() {
+        selectInterestsViewController.userIsFollowingTopics { [unowned self] isFollowing in
+            if isFollowing {
+                self.hideSelectInterestsView()
+            } else {
+                self.showSelectInterestsView()
+            }
+        }
+    }
+
+    func hideSelectInterestsView() {
+        guard selectInterestsViewController.parent != nil else {
+            if shouldForceRefresh {
+                displayLoadingStream()
+                syncIfAppropriate(forceSync: true)
```

Comment:
```suggestion
                super.syncIfAppropriate(forceSync: true)
```

Otherwise, this won't work if the user has scrolled. If this is the intended behavior, no worries tho. :)

Reply:
Good point! Done here: adf567047e30156be0e515ba3a52f250dee27a5c